### ISSUE TITLE
no version information available

### ISSUE THREAD

**Issue by ghkareem** [kobolabs/liblzma#3]
Hi, I tried to instlall with 
./configure --enable-shared --prefix=<non standard location>
and I can compile, but when I try to call
xz --version
I get the following error:

 no version information available (required by xz)
xz: symbol lookup error: xz: undefined symbol: lzma_stream_encoder_mt_memusage, version XZ_5.2

Thanks for any help!

**Comment by BlackMage2** [comment]
yes in the lib is missing version information and some functions in xz lib, for example apt wont work with it: `/usr/lib/apt/methods/http: /usr/local/lib/liblzma.so.5: no version information available (required by /lib/x86_64-linux-gnu/libsystemd.so.0` and some other xz and lzma info functions are even not available

for example apt-extracttemplate use the lib, version information should be added, so that these messages disappear

**Comment by BlackMage2** [comment]
and for example bsdtar writes: `bsdtar: symbol lookup error: /lib/x86_64-linux-gnu/libarchive.so.13: undefined symbol: lzma_stream_encoder_mt, version XZ_5.2` (almost exactly the same message as the issue creator.)
